rlkit/torch/networks/ae_tanh_policy.py

```diff
@@ -0,0 +1,35 @@
+import torch
+
+from rlkit.torch import pytorch_util as ptu
+from rlkit.torch.networks import MlpPolicy
+
+
+class AETanhPolicy(MlpPolicy):
```

Comment:
we could also delete this if no one is using it

Reply:
deleted